File: java/src/main/java/com/dreizak/miniball/highdim/Logging.java
package com.dreizak.miniball.highdim;

/**
 * A very simple logging class used for debugging.
 */
class Logging
{
  final static boolean log = false;

  public final static void warn(String msg)
  {
    if (log) System.err.println("[warn] " + msg);
  }

  public final static void info(String msg)
  {
    if (log) System.err.println("[info] " + msg);
  }

  public final static void debug(String msg)
  {
    if (log) System.err.println("[debug] " + msg);
  }
}

File: csharp/miniball/highdim/Logging.cs
namespace SEB;

/// <summary>
/// A very simple logging class used for debugging.
/// </summary>
static class Logging
{

    public static bool log = false;

    public static void Warn(string msg)
    {
        if (log) System.Console.WriteLine($"[warn] {msg}");
    }

    public static void Info(string msg)
    {
        if (log) System.Console.WriteLine($"[info] {msg}");
    }

    public static void Debug(string msg)
    {
        if (log) System.Console.WriteLine($"[debug] {msg}");
    }

}